de/events.json

```diff
@@ -1 +1,27 @@
-{}
+{
+  "page-title": "Nächste Events ({{platform}}) - Eric's GoW Tools",
+  "page-desc": "Liste von zukünftigen Events in Gems of War, täglich aktualisiert.",
+
+  "head-name": "Name",
+  "head-start-date": "Startdati,",
```

Comment:
I'd go for singular "Startdatum" here instead of plural "Startdati".... even though it's a list/column of dates, every event just has one single start date.